pesacheck_meedan_bridge/py/main.py

```diff
@@ -57,14 +68,16 @@ def post_to_check_and_update(feed, db):
         if language.lower() in language_codes
     ]
     language = "en" if not codes else codes[0]
+    claim_description = feed.title
+    summary = extract_summary(feed.description) or "Not Found"
     input_data = {
         "media_type": "Blank",
         "channel": 1,
         "set_tags": categories,
         "set_status": "verified",
-        "set_claim_description": f"""{html_to_text(feed.description)}""",
+        "set_claim_description": f"""{claim_description}""",
```

Comment:
Add a comment line why we need `f"""`. My understanding `claim_description`, `summary`, etc. are already strings so why do we need `f"""`?